rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "stable"
+components = ["rustfmt"]
+profile = "minimal"
```

Comment:
The `minimial` profile doesn't include `clippy`, so add that to components explicitly.

https://rust-lang.github.io/rustup/concepts/profiles.html

Reply:
In the `clippy.yml` workflow we ask to install clippy throgh:
```
- name: Setup Rust
        uses: './.github/actions/rust/rust-setup'
        with:
          os: ${{ runner.os }}
          job: ${{ github.job }}
          target: wasm32-unknown-unknown
          components: clippy
```